examples/linear_model/plot_poisson_regression_non_normal_loss.py

```diff
@@ -1,3 +1,5 @@
+# Authors: The scikit-learn developers
+# License: BSD 3 clause
```

Comment:
I think going with this suggestion makes sense: https://github.com/scikit-learn/scikit-learn/issues/20813#issuecomment-2075621386

```suggestion
# SPDX-License-Identifier: BSD-3-Clause
```

Reply:
I don't mind, then we can update the blog for this as well.